definitions/json/CaseEvent/CaseEvent-HMC-nonprod.json

```diff
@@ -13,7 +13,7 @@
     "EventEnablingCondition": "managingOffice !=\"Unassigned\"",
     "CallBackURLAboutToStartEvent": "",
     "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/addAmendClaimantRepresentative/aboutToSubmit",
-    "CallBackURLSubmittedEvent": ""
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/amendRespondentRepSubmitted"
```

Comment:
Would this be a little confusing where the event is for Claimant reps but the submitted url is respondent rep?